index.html

```diff
@@ -65,7 +65,17 @@ <h6>Tonic</h6>
           <li><a id="language-name" href="#">css</a></li>
           <li><a id="language-name" href="#">javascript</a></li>
         </ul>
-        <div><a class="see-projects" href="#">See Projects</a></div>
+        <div><a class="see-projects"  id="modalbtn1">See Projects</a></div>
+           <div class="modalContainer1">
+           <div class="modal1" id="modal1">
+              <h1>project deatail 1 </h1>
+               <p> mobile-vertion</p>
+               <p> desktop-vertion</p>
+               <p> mobile-menu</p>
+               <button id="closemodal1" class="button">close</button>
+
+           </div>
+          </div>
       </div>
```

Comment:
- [ ] Please use a JavaScript array to store all of the information for all projects. (For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.)
  - Please make sure that the model is created dynamically using javascript, not in the `index.html` file. You should just use an empty container in the `index.html` file that should populate the model dynamically with the information stored in the javascript array. `(With this approach you will not repeat the code, A single container can be used for all the project models)`.